MapboxVisionAR/Render/ARScene/ConcreteNodes/ARLaneNode/ARLaneMesh/ARLaneMesh.swift

```diff
@@ -0,0 +1,38 @@
+import MetalKit
+
+class ARLaneMesh {
```

Comment:
The only purpose of the class is to load a mesh with name "lane", right? Maybe it's better to use pure function?

Reply:
Yeah, as I said there's a lot of places to improve, but currently implementation of ARLaneMesh depends on vertex descriptors and current Metal device. Thus if we go to change the structure of code I'd prefer to refactor ARRenderer, but I believe that's not a part of current scope.

Reply:
Agree to get rid of class and use a function.